### ISSUE TITLE
Triton Provider Tutorial/Documentation is incorrect

### ISSUE THREAD

**Issue by sodre** [hashicorp/terraform#6075]
If you follow the [Triton Provider documentation](https://www.terraform.io/docs/providers/triton/index.html)  you end up with the following error 
```
$ terraform plan
There are warnings and/or errors related to your configuration. Please
fix these before continuing.

Errors:

  * provider.triton: : invalid or unknown key: key_path
```

The code uses `key_material` instead of key, or key_path. e70764f64dc23e2d26a0f53d45b019fde325188c

**Comment by stack72** [comment]
Hi @sodre 

Thanks for reporting this. apologies for the incorrect documentation. I have fixed this in #6090 and this will be merged as soon as it goes green :)

Paul

**Comment by hkle2000** [comment]
I think It still remains
